config/configmap/inferenceservice.yaml

```diff
@@ -41,7 +41,38 @@ data:
                "defaultImageVersion": "latest"
            }
        }
-     
+    # ====================================== ISVC CONFIGURATION ======================================
+    # Example   
+     inferenceservice: |-
+      {
+        "serviceAnnotationDisallowedList": [
+          "my.custom.annotation/1"  
+        ],
+        "serviceLabelDisallowedList": [
+          "my.custom.label.1"  
+        ]
+       }
+    # Example of isvc configuration
+    inferenceservice: |-
+      {
+        # ServiceAnnotationDisallowedList is a list of annotations that are not allowed to be propagated to Knative 
```

Comment:
This is only applicable for Knative revisions?

Reply:
Yes